### PR TITLE
Add documentation for the PSA sub-command

### LINKED ISSUE #3: add documentation (for PSA)
add documentation for the sub-commands currently implemented (i.e., psa)

### PR DESCRIPTION
**Pull request by thomas-fossati**
The help is rendered [here](https://github.com/veraison/evcli/blob/docs/README.md).

Fix #3

Signed-off-by: Thomas Fossati <thomas.fossati@arm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,170 @@
...
+    --claims=psa-claims-profile-1.json \
+    --key=ec256.json \
+    --profile=PSA_IOT_PROFILE_1
+```
+
+### Check
+
+Use the `psa check` subcommand to verify the cryptographic signature over the supplied PSA attestation token as well as checking whether its claim set is well-formed.
+
+To check the PSA attestation token in my.cbor using the public key in es256.jwk:
```
Comment (SimonFrost-Arm) [inline README.md:75]: Typo: .jwk vs .json in the example

**Review by SimonFrost-Arm — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (thomas-fossati) [inline README.md:75]:
```suggestion
To check the PSA attestation token in my.cbor using the public key in es256.json:
```

File: README.md [hunk 2]
```diff
@@ -1 +1,170 @@
...
+A message will indicate whether the signature has been successfully verified:
+
+```console
+>> "my.cbor" verified
+```
+
+In such case, the claim set is printed to stdout in JSON format:
+
+```json
+>> embedded claims:
```
Comment (yogeshbdeshpande) [inline README.md:93]: Seems line 93 does not come up nicely in the redered .md also, needs correction on this one!

File: README.md [hunk 3]
```diff
@@ -1 +1,170 @@
...
+```shell
+evcli psa verify-as attester \
+    --api-server=https://veraison.example/challenge-response/v1 \
+    --claims=psa-claims-profile-2-without-nonce.json \
+    --key=es256.json
+```
+
+Note that the supplied claims file must not include a nonce claim.
+
+By default, the command will request 48 bytes nonce from the server.  If needed, a different value can be requested using the `--nonce-size` (abbrev. `-n`) switch.  Available nonce sizes are 32, 48 or 64 bytes, as per PSA attestation token specification.
```
Comment (yogeshbdeshpande) [inline README.md:149]: A link to PSA attestation token specification will certainly help here.

File: README.md [hunk 4]
```diff
@@ -1 +1,170 @@
...
+    --token=my.cbor \
+    --key=es256.json \
+    --claims=output-claims.json
+```
+
+### Verify
+
+The `psa verify-as` subcommand allows you to interact with the Veraison Verifier (or another Attestation Verifier implementing the Veraison [challenge-response API](https://github.com/veraison/docs/tree/main/api/challenge-response)).
+
+There are two modes of operation corresponding to the emulated roles: [Attester](#attester) or [Relying Party](#relying-party).
```
Comment (yogeshbdeshpande) [inline README.md:134]:
```suggestion
There are two modes of operation corresponding to the emulated roles: [Attester](#attester) or [Relying Party](#relying-party), for interaction with a Verifier as documented in [Remote Attestation Procedures Architecture](https://datatracker.ietf.org/doc/html/draft-ietf-rats-architecture-15)
```

**Review by yogeshbdeshpande — APPROVED** [review]
LGTM! Left few minor comments for consideration.

File: README.md (hunk 2 above)
Comment (thomas-fossati) [reply README.md:93]: > Seems line 93 does not come up nicely in the redered .md

yes, JSON highlighting is not very tolerant :-)

> also, needs correction on this one!

I don't understand what needs correcting?

File: README.md (hunk 4 above)
Comment (thomas-fossati) [reply README.md:134]: ok

File: README.md (hunk 3 above)
Comment (thomas-fossati) [reply README.md:149]: ok

**Review by yogeshbdeshpande — APPROVED** [review]

**Pull request merged**